composer.json

```diff
@@ -88,7 +75,6 @@
 		"roots/wordpress": "^6.0",
 		"wp-cli/wp-cli": "^2.6",
 		"wp-phpunit/wp-phpunit": "^6.1",
-		"wp-premium/woocommerce-subscriptions": "^3.0",
```

Comment:
Is this necessary? Removes the ability to navigate through WooCommerce Subscriptin functions in editor 😢

Reply:
Re-added latest version in a932d1b662d4e4a9151ed55dd5d52e15f4f40a15.